TestProject/TestProject/UnitTest1.cs

```diff
@@ -0,0 +1,32 @@
+using System.Threading;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace TestProject;
+
+public class Tests
+{
+    private IWebDriver driver;
+    private readonly string HOME_PAGE_URL = "https://vadimevteev.testrail.io/";
```

Comment:
Create `appsettings.json` and store all configuration data

https://weblog.west-wind.com/posts/2018/Feb/18/Accessing-Configuration-in-NET-Core-Test-Projects

Reply:
Also as I remember `HOME_PAGE_URL` - is not correct C# code convention